.docker-images/windows-servercore-ltsc2022-micromamba/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
+USER ContainerAdministrator
+RUN powershell -Command Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://vcredist.com/install.ps1'))
+USER ContainerUser
+ADD https://github.com/mamba-org/micromamba-releases/releases/download/1.5.10-0/micromamba-win-64 micromamba.exe
+RUN mkdir %USERPROFILE%\micromamba && \
+    copy micromamba.exe %USERPROFILE%\micromamba && \
+    del micromamba.exe && \
+    powershell -Command \
+        [Environment]::SetEnvironmentVariable('PATH', $env:Path + ';' + $env:USERPROFILE + '\micromamba', 'User') && \
+    powershell -Command \
+        [Environment]::SetEnvironmentVariable('MAMBA_ROOT_PREFIX', $env:USERPROFILE + '\micromambaenv', 'User')
+RUN micromamba shell init -s powershell && \
+    micromamba config append channels conda-forge && \
+    micromamba config append channels nodefaults && \
+    micromamba config set channel_priority strict
```

Comment:
It's the usual way micromamba is configured when an user download it online. But, could you sent me the link to the code of Analyst installation ? I don't found it. 🧐

Reply:
basically, analyst does nothing. No config step or anything.
Furthermore, from MambaEnvRunner:         `Environment.SetEnvironmentVariable("CONDARC", _installDir);`
pointing rc file to an empty folder so that it uses no rc file, but still gives room for having a dedicated custom config (to help with proxy maybe).

Since there is no rc file to read channels from, channels are passed through env variables by Analyst: `CONDA_CHANNELS` (coma-separated list, according to [conda doc]( https://docs.conda.io/projects/conda/en/24.9.x/configuration.html), and tested)

Reply:
some reference to Analyst code:
- https://github.com/MiraGeoscience/InSight/blob/f6aeac23d9691cd50e0d102ec31681aaca12c493/src/Base/Interfaces/MiraPyPackageIndex.cpp#L71
- https://github.com/MiraGeoscience/InSight/blob/f6aeac23d9691cd50e0d102ec31681aaca12c493/src/Base/Interfaces/InternalPythonEnvironment.cpp#L501
- https://github.com/MiraGeoscience/CmdRunner/blob/develop/src/MambaEnvInstaller/MambaEnvRunner.cs#L359

Reply:
Thx